openrouteservice/src/test/java/org/heigit/ors/routing/graphhopper/extensions/weightings/AdditionWeightingTest.java

```diff
@@ -0,0 +1,33 @@
+package org.heigit.ors.routing.graphhopper.extensions.weightings;
+
+import com.graphhopper.routing.util.EncodingManager;
+import org.heigit.ors.routing.graphhopper.extensions.ORSDefaultFlagEncoderFactory;
+import org.heigit.ors.routing.graphhopper.extensions.flagencoders.CarFlagEncoder;
+import org.heigit.ors.routing.graphhopper.extensions.flagencoders.FlagEncoderNames;
+import org.heigit.ors.routing.graphhopper.extensions.weighting.AdditionWeighting;
+import org.heigit.ors.routing.graphhopper.extensions.weighting.ConstantWeighting;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class AdditionWeightingTest {
+    private final EncodingManager encodingManager;
+    private final CarFlagEncoder flagEncoder;
+
+    public AdditionWeightingTest() {
+        encodingManager = EncodingManager.create(new ORSDefaultFlagEncoderFactory(), FlagEncoderNames.CAR_ORS, 4);
+        flagEncoder = (CarFlagEncoder)encodingManager.getEncoder(FlagEncoderNames.CAR_ORS);
+    }
+
+    @Test
+    public void sumOfConstants () {
+        ConstantWeighting const1 = new ConstantWeighting(1,10);
+        ConstantWeighting const2 = new ConstantWeighting(2,20);
+        ConstantWeighting const3 = new ConstantWeighting(3,30);
+        ConstantWeighting factor = new ConstantWeighting(10, 100);
```

Comment:
I know it's just a name, yet it confused me at first that the super weighting is called factor in this context..

Reply:
You are right. The name "superWeighting" didn't seem to make sense in the previous implementation. Now that we clarified that "calcMillis" was wrong and what its true purpose is, it makes sense again.